scripts/create_test

```diff
@@ -104,32 +111,59 @@ OR
                         "all object executables, and data files will"
                         " be removed after tests are run")
 
-    parser.add_argument("-c", "--compare", const=True, nargs="?",
-                        help="While testing, compare baselines,"
-                        " optionally provide a compare directory ")
+    if model == "cesm":
+        parser.add_argument("-c", "--compare",
+                            help="While testing, compare baselines"
+                            "  against the given compare directory ")
+
+        parser.add_argument("-g", "--generate",
+                            help="While testing, generate baselines,"
+                            " to the given generate directory")
+
+        parser.add_argument("--xml-machine",
+                            help="Use this machine key in the lookup in testlist.xml, default is all if any --xml- argument is used")
+
+        parser.add_argument("--xml-compiler",
+                            help="Use this compiler key in the lookup in testlist.xml, default is all if any --xml- argument is used")
+
+        parser.add_argument("--xml-category",
+                            help="Use this category key in the lookup in testlist.xml, default is all if any --xml- argument is used")
+
+        parser.add_argument("--xml-testlist",
+                            help="Use this testlist to lookup tests, default specified in config_files.xml")
+
+        parser.add_argument("testargs", nargs="*",
+                            help="Tests or test suites to run."
+                            " Testnames expect in form CASE.GRID.COMPSET[.MACHINE_COMPILER]")
+
+        parser.add_argument("-m", "--machine",
+                            help="The machine for which to build tests, this machine must be defined"
+                            " in the config_machines.xml file for the given model. "
+                            "Default is to match the name of the machine in the test name or "
+                            "the name of the machine this script is run on to the "
+                            "NODENAME_REGEX field in config_machines.xml")
+
+    else:
 
-    parser.add_argument("-g", "--generate", const=True, nargs="?",
-                        help="While testing, generate baselines,"
-                        " optionally provide a generate directory")
+        parser.add_argument("testargs", nargs="+",
```

Comment:
You guys still have it, the difference is that it's a "*" for you guys (optional)